notebooks/00_quick_start/rbm_movielens.ipynb

```diff
@@ -0,0 +1,885 @@
+    "                               col_rating=\"rating\", col_prediction=\"prediction\", \n",
+    "                               relevancy_method=\"top_k\", k= K)\n",
+    "\n",
+    "    eval_recall = recall_at_k(data_true, data_pred, col_user=\"userID\", col_item=\"movieID\", \n",
+    "                          col_rating=\"rating\", col_prediction=\"prediction\", \n",
+    "                          relevancy_method=\"top_k\", k= K)\n",
+    "\n",
+    "    \n",
+    "    df_result = pd.DataFrame(\n",
+    "        {   \"Dataset\": data_size,\n",
+    "            \"K\": K,\n",
+    "            \"MAP\": eval_map,\n",
+    "            \"nDCG@k\": eval_ndcg,\n",
+    "            \"Precision@k\": eval_precision,\n",
+    "            \"Recall@k\": eval_recall,\n",
+    "            \"Train time\": time_train,\n",
+    "            \"Test time\": time_test\n",
+    "        }, \n",
+    "        index=[0]\n",
+    "    )\n",
+    "    \n",
+    "    return df_result"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 14,
+   "metadata": {
+    "scrolled": true
+   },
+   "outputs": [
+    {
+     "data": {
+      "text/html": [
+       "<div>\n",
+       "<style scoped>\n",
+       "    .dataframe tbody tr th:only-of-type {\n",
+       "        vertical-align: middle;\n",
+       "    }\n",
+       "\n",
+       "    .dataframe tbody tr th {\n",
+       "        vertical-align: top;\n",
+       "    }\n",
+       "\n",
+       "    .dataframe thead th {\n",
+       "        text-align: right;\n",
+       "    }\n",
+       "</style>\n",
+       "<table border=\"1\" class=\"dataframe\">\n",
+       "  <thead>\n",
+       "    <tr style=\"text-align: right;\">\n",
+       "      <th></th>\n",
+       "      <th>Dataset</th>\n",
+       "      <th>K</th>\n",
+       "      <th>MAP</th>\n",
+       "      <th>nDCG@k</th>\n",
+       "      <th>Precision@k</th>\n",
+       "      <th>Recall@k</th>\n",
+       "      <th>Train time</th>\n",
+       "      <th>Test time</th>\n",
+       "    </tr>\n",
+       "  </thead>\n",
+       "  <tbody>\n",
+       "    <tr>\n",
+       "      <th>0</th>\n",
+       "      <td>mv 100k</td>\n",
+       "      <td>10</td>\n",
+       "      <td>0.208581</td>\n",
+       "      <td>0.563283</td>\n",
+       "      <td>0.472853</td>\n",
+       "      <td>0.275245</td>\n",
+       "      <td>1.713896</td>\n",
+       "      <td>0.153406</td>\n",
+       "    </tr>\n",
+       "  </tbody>\n",
+       "</table>\n",
+       "</div>"
+      ],
+      "text/plain": [
+       "   Dataset   K       MAP    nDCG@k  Precision@k  Recall@k  Train time  \\\n",
+       "0  mv 100k  10  0.208581  0.563283     0.472853  0.275245    1.713896   \n",
+       "\n",
+       "   Test time  \n",
+       "0   0.153406  "
+      ]
+     },
+     "execution_count": 14,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "eval_100k= ranking_metrics(\n",
```

Comment:
It would be good to use `papermill` to record the evaluation results for testing purpose. See [SAR](https://github.com/Microsoft/Recommenders/blob/master/notebooks/00_quick_start/sar_single_node_movielens.ipynb) for example.